boost/CMakeLists.txt

```diff
@@ -6,7 +6,8 @@ if( UNIX )
     set( boost_CMDbootstrap bootstrap.sh )
     set( boost_CMDb2 b2 )
 else()
-    message(FATAL_ERROR "boost building implemented only on unix")
+    message(WARNING "boost building implemented only on unix")
```

Comment:
Could you elaborate, why you don't like FATAL_ERROR here?

Reply:
FATAL_ERROR makes the build stop.
On Windows I just want the user to be informed that boost will not be used.